docs/operations/tap.md

```diff
@@ -1,103 +1,289 @@
 ---
-title: "Topology-Aware Placement (TAP)"
-linkTitle: Topology-Aware Placement (TAP)
+title: "How To Enable Topology-Aware Placement (TAP)"
+linkTitle: How To Enable Topology-Aware Placement (TAP)
 ---
 
-Ondat Topology-Aware Placement is a feature that enforces placement of data
-across failure domains to guarantee high availability.
+## Overview
 
-TAP uses default labels on nodes to define failure domains. For instance, an
-Availability Zone. However, the key label used to segment failure domains can
-be defined by the user per node. Also, TAP is an opt in feature per volume. The
-following guide demonstrates how to use topology-aware placement. We will
-enable Topology-Aware Placement on a PVC, set it to the `soft` Failure Mode.
+Ondat Topology-Aware Placement is a feature that enforces placement of data across failure domains to guarantee high availability.
+- TAP uses default labels on nodes to define failure domains. For instance, an Availability Zone. However, the key label used to segment failure domains can be defined by the user per node. In addition, TAP is an opt-in feature per volume. 
 
-## HowTo enable TAP per volume
+> 💡 For more information on the Ondat Topology-Aware Placement feature, review the [Ondat Topology-Aware Placement](/docs/concepts/tap) feature page.
 
-> 💡 Labels can be applied to a PVC directly, or indirectly by adding them
-> as parameters on a StorageClass.
+### Example - Enable Topology-Aware Placement Through a `PersistentVolumeClaim` Definition
 
-1. Ensure that all nodes in your cluster have the topology key set. Here we
-   will set a custom node zone label.
+The following guidance below will demonstrates how to use Ondat Topology-Aware Placement through a `PersistentVolumeClaim` (PVC) definition. 
+- The instructions will enable Topology-Aware Placement on a PVC, use a custom zone labelling scheme with the label >> `storageos.com/topology-key=custom-region` and set it to the `soft` Failure Mode.
```

Comment:
is the `>>` intended?

Reply:
yes - I was use it point to specific items, but I am happy to remove it

Reply:
lets keep it